src/main/webapp/assets/modules/get-promoted/userPromotionSurveyFormDirective.js

```diff
@@ -0,0 +1,12 @@
+techlooper.directive("userPromotionSurveyForm", function () {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/get-promoted/userPromotionSurveyForm.html",
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.